### PR TITLE
Decoder pose

### PR DESCRIPTION
**Pull request by xroumegue**
Hi nnstreamer folks,

This PR adds support for using pose decoder with latest posenet neural network model which are using 17 different key points.

https://www.tensorflow.org/lite/examples/pose_estimation/overview#model_description

A sub plugin pose decoder option(3) has been added to specify the body joint labels.
If the option is not set or the specified file is not present, the default (14) pose labels are used as fallback.

Signed-off-by: Xavier Roumegue <xavier.roumegue@nxp.com>

Self evaluation:
Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by again4you — APPROVED** [review]

**Comment by again4you** [comment]
First of all, thank you for your interest and PR.

The patch you made looks good for me. I will give you +1. In my humble opinion, the default shown label might make people a little bit confusing. I do not talk about this issue with my team members yet. Probably Someone will give you their opinion.

**Comment by anyj0527** [comment]
@xroumegue 
Currently, the decoder is written for specific tflite model (it's quite outdated). The `draw (...)` part is only for the model. It draws lines among 14 body points in hardcoded way.

I think you may rewrite this `draw` function for the current standard model `posenet_mobilenet_v1_100_257x257_multi_kpt_stripped.tflite`.
or
Add feature which supports option for drawing lines (among the body points which line should be drawn) besides label info.

**Comment by xroumegue** [comment]
Thanks @again4you @anyj0527 for your comments.

Indeed, draw() has to be modified accordingly to the body joints model. Thanks for catching this.
I will amend the PR accordingly, without hard coding.

Do you want to use the 17 body points model as default ?

**Review by anyj0527 — APPROVED** [review]
@xroumegue

> Do you want to use the 17 body points model as default ?

No. If you want to support the 17 body points model, you may add a new `mode` for it in a new PR.
(REF for `mode`: [tensordec-imagesegment.c](https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c) )

The 17 body points model has four output tensors and the new mode should deal with these output tensors.
You may refer this example using the model: https://github.com/nnstreamer/nnstreamer-example/tree/main/native/example_pose_estimation_tflite

In the new PR `draw()` can be configured by label and additional line info without hardcoding.

Anyway, this PR seems valid. Thanks!

**Comment by xroumegue** [comment]
> @xroumegue
> 
> > Do you want to use the 17 body points model as default ?
> 
> No. If you want to support the 17 body points model, you may add a new `mode` for it in a new PR.
> (REF for `mode`: [tensordec-imagesegment.c](https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c) )
> 
> The 17 body points model has four output tensors and the new mode should deal with these output tensors.
> You may refer this example using the model: https://github.com/nnstreamer/nnstreamer-example/tree/main/native/example_pose_estimation_tflite
Sure. Will do in the coming days. It was part of the plan :)

> 
> In the new PR `draw()` can be configured by label and additional line info without hardcoding.
Will do as well !

> 
> Anyway, this PR seems valid. Thanks!

**Review by myungjoo — APPROVED** [review]
LGTM
